data/projects.js

```diff
@@ -230,4 +230,45 @@ implemented.",
 		"initiatives" : ["GSoC"],
 		"tags" : ["Core", "Bears", "Debugging", "pdb"]
 	}
+	{
+		"name" : "Handle languages inside other languages",
+		"desc" : "Multiple programming languages can coexist in a single \
+source file and coala would benefit from having an API to support such \
+analysis algorithms. If we have a php file, it is inevitable that we'd have \
+chunks of html inside a few of them. We want some method of finding specific \
+languages inside other languages and handle them accordingly. Like awk inside \
+bash, JSON inside Javascript, etc.",
```

Comment:
what kind of sentence is that `Like awk inside bash, JSON inside Javascript, etc.`